scripts/fix-image-policies.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
better to prefer `/usr/bin/env bash` so it loads bash from the path rather than hardcoding, especially better on macOS (for testing) that has frozen to a very old version of bash